assets/dev/js/editor/elements/views/base.js

```diff
@@ -841,13 +844,9 @@ BaseElementView = BaseContainer.extend( {
 			return;
 		}
 
-		if ( options.scrollIntoView ) {
```

Comment:
Why?

Reply:
Check out [this](https://github.com/elementor/elementor/pull/17937/files/24c6737e3856d4e105ba901845e44907832b1e52#diff-1112993f94abb738bc19f7ea5baaa313c82feefda8405861d58a423502cb5d6dR10) change.